SPECS/xapi.spec

```diff
@@ -62,34 +70,27 @@ Patch1: 0001-Xen-4.19-domctl_create_config.vmtrace_buf_kb.patch
 Patch2: 0002-Xen-4.20-domctl_create_config.altp2m_ops.patch
 %endif
 
-# XCP-ng patches
+# Backports:
```

Comment:
This is supposed to be `# XCP-ng patches`, as was the case before. Plus, `Backports` is not an accurate name for most of the patches below.